src/apps/experimental/components/AppToolbar/index.tsx

```diff
@@ -24,11 +24,16 @@ const ExperimentalAppToolbar: FC<AppToolbarProps> = ({
     onDrawerButtonClick
 }) => {
     const location = useLocation();
+
+    // The video osd does not show the standard toolbar
+    if (location.pathname === '/video') return null;
+
     const isTabsAvailable = isTabPath(location.pathname);
+    const isPublicPath = location.pathname === '/selectserver.html';
```

Comment:
Should `addserver` also be listed as a public path?

Reply:
We probably _could_ check all public paths... the select server screen was particularly an issue because choosing "select server" does not logout the user.

Reply:
:+1: updated to include all public pages (excluding the wizard).